examples/otel-logs-routing/docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: "3"
+services:
+  # Sample Go application producing logs.
+  logging:
+    image: bash:latest
+    container_name: logging
+    command: 'bash -c "while(true) do echo \"$${date} new message\" >> /output/file.log ; sleep 1; done"'
+    volumes:
+      - ./output:/output
+  # Splunk Enterprise server:
+  splunk:
+    image: splunk/splunk:latest
+    container_name: splunk
+    environment:
+      - SPLUNK_START_ARGS=--accept-license
+      - SPLUNK_HEC_TOKEN=00000000-0000-0000-0000-0000000000000
+      - SPLUNK_PASSWORD=changeme
+    ports:
+      - 18000:8000
+    healthcheck:
+      test: ['CMD', 'curl', '-f', 'http://localhost:8000']
+      interval: 5s
+      timeout: 5s
+      retries: 20
+    volumes:
+      - ./splunk.yml:/tmp/defaults/default.yml
+      - /opt/splunk/var
+      - /opt/splunk/etc
+  # OpenTelemetry Collector
+  otelcollector:
+    image:  quay.io/signalfx/splunk-otel-collector:0.29.0
```

Comment:
Any reason not to use latest or a more recent build?

Reply:
Copied from another sample. I assumed there was a job updating all samples or something in place around that and therefore a good reason for the version used.